src/lib.rs

```diff
@@ -64,8 +74,26 @@ pub fn drop_table(table: &str) -> io::Result<()> {
     Ok(())
 }
 
+#[allow(unused_must_use)]
+pub fn append_records<T: Serialize + Deserialize>(table: &str, t: T) -> io::Result<()> {
+    let mut data: Data<T>     = serde_json::from_str(&read_table("test")).unwrap();
+    let     increased_next_id = data.next_id.parse::<i32>().unwrap();
+    let     new_id            = increased_next_id + 1;
+
+    data.records.insert(increased_next_id.to_string(), t);
+    data.next_id = new_id.to_string();
+
+    upgrade_table(table, &data)
+}
+
 // private functions and tests
 
+#[allow(dead_code, unused_must_use)]
```

Comment:
Well this is a private function. I would not want a user to be able to use this if they did not know what they were doing.